homeassistant/components/zwave_js/trigger.py

```diff
@@ -2,45 +2,13 @@
 
 from __future__ import annotations
 
-from homeassistant.const import CONF_PLATFORM
-from homeassistant.core import CALLBACK_TYPE, HomeAssistant
-from homeassistant.helpers.trigger import (
-    TriggerActionType,
-    TriggerInfo,
-    TriggerProtocol,
-)
-from homeassistant.helpers.typing import ConfigType
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.trigger import Trigger
+from homeassistant.util.decorator import Registry
 
-from .triggers import event, value_updated
```

Comment:
I think nothing imports the `event.py` module anymore. That means the `event` trigger won't be registered in `TRIGGERS`.

Reply:
Yeah, that's right. This means the registry approach doesn't work.